alert-database/src/main/java/com/synopsys/integration/alert/database/job/DistributionJobRepository.java

```diff
@@ -61,8 +61,9 @@
                        + "            OR policyFilters.policyName IN (:policyNames)"
                        + "          )"
                        + "    )"
+                       + "GROUP BY jobEntity.jobId ORDER BY jobEntity.createdAt ASC"
     )
-    Page<DistributionJobEntity> findMatchingEnabledJobsByFilteredNotifications(
+    Page<DistributionJobEntity> findMatchingEnabledJobsByFilteredNotificationsGroupByJobIdOrderByCreatedAt(
```

Comment:
Because we use a custom query, I don't think this method name being this verbose is necessary. Perhaps something like `findAndSortMatchingEnabledJobsByFilteredNotifications` (which is already pretty long).